packages/shared/src/components/search/SearchContainer.tsx

```diff
@@ -17,7 +17,7 @@ export function SearchContainer({
     <main className="m-auto w-full">
       <div className="grid grid-cols-1 laptop:grid-cols-3 gap-y-6 py-8 m-auto w-full max-w-screen-laptopL">
         <main className="flex flex-col flex-1 col-span-2 px-4 laptop:px-8">
-          <SearchBar onSubmit={onSubmit} chunk={chunk} showProgress />
+          <SearchBar onSubmit={onSubmit} chunk={chunk} showProgress={!!chunk} />
```

Comment:
It fills clunky that `!!chunk` is a condition here for `showProgress`, I expected `isLoading` here but it becomes `false`when search ends which then does not show final `Done` status label.

Reply:
Hmm maybe a non urgent improvement we can backlog?
Technically it can probably get the data rather from a wrapping hook, instead of passing it down anyway